examples/btc-to-evm-fungible-transfer/src/blockstreamApi.ts

```diff
@@ -0,0 +1,171 @@
+import { TypeOfAddress } from "@buildwithsygma/bitcoin";
+import type { BitcoinTransferParams } from "@buildwithsygma/bitcoin";
+import type { Network, Signer } from "bitcoinjs-lib";
+import { payments, Psbt } from "bitcoinjs-lib";
+
+type Utxo = {
+  txid: string;
+  vout: number;
+  status: {
+    confirmed: boolean;
+    block_height: number;
+    block_hash: string;
+    block_time: number;
+  };
+  value: number;
+};
+
+type FeeEstimates = Record<string, number>;
+
+export async function getFeeEstimates(blockstreamUrl: string): Promise<number> {
+  if (!blockstreamUrl) throw new Error("Blockstream url is required");
+  try {
+    const response = await fetch(`${blockstreamUrl}/fee-estimates`);
+    const BLOCK_CONFIRMATION_INDEX = "5";
```

Comment:
As it's a parameter that user might want to configure we can move it to the top of the file or as env variable

Reply:
😄 @saadjhk differs in this approach

Reply:
yes I asked @wainola to move this within the function to avoid confusion as this constant is only relevant for this function. I think having it defined at the top of the file and env variable is not needed, at it is very specific to this function.

Reply:
will resolve this as I think in this case @saadjhk argument makes more sense

Reply:
possible solution to this could be to add a new env var for this and just use that, wdty @saadjhk @Lykhoyda ?

Reply:
Even if it's used inside the method, it is a config variable that influences how the transaction will be approved, so it's important to show the user the possibility of changing that behavior.  That's why to make it more visible both options work for me put it on top or create Env variable

Reply:
lets keep it at the top